tests/FormBuilderTest.php

```diff
@@ -17,7 +17,8 @@ public function setUp()
   {
     $this->urlGenerator = new UrlGenerator(new RouteCollection, Request::create('/foo', 'GET'));
     $this->htmlBuilder = new HtmlBuilder($this->urlGenerator);
-    $this->formBuilder =  new FormBuilder($this->htmlBuilder, $this->urlGenerator, 'abc');
+    $this->router = m::mock('Illuminate\Routing\RouteCollection');
+    $this->formBuilder =  new FormBuilder($this->htmlBuilder, $this->urlGenerator, $this->router, 'abc');
   }
```

Comment:
Here I see you updated the tests so that existing functionality wouldn't break. 

Can you create tests to cover the added functionality offered by this PR?